Deprecation warning in `test_merge_sorted_of_proxied_cudf_dataframes`

There is a cuDF deprecation warning from `merge_sorted` while running `test_merge_sorted_of_proxied_cudf_dataframes`, see details below:

<details>

```python
=================================== FAILURES ===================================
_________________ test_merge_sorted_of_proxied_cudf_dataframes _________________

    def test_merge_sorted_of_proxied_cudf_dataframes():
        cudf = pytest.importorskip("cudf")

        dfs = [cudf.DataFrame({"a": range(10)}), cudf.DataFrame({"b": range(10)})]
>       got = cudf.merge_sorted(proxify_device_objects(dfs, {}, []))

dask_cuda/tests/test_proxy.py:583:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

objs = [<dask_cuda.proxy_object.ProxyObject at 0x7f4ef6bb65b0 of cudf.core.dataframe.DataFrame at 0x7f4dd54fb070>, <dask_cuda.proxy_object.ProxyObject at 0x7f4ef6bb67f0 of cudf.core.dataframe.DataFrame at 0x7f4ef70b2a30>]
keys = None, by_index = False, ignore_index = False, ascending = True
na_position = 'last'

    def merge_sorted(
        objs,
        keys=None,
        by_index=False,
        ignore_index=False,
        ascending=True,
        na_position="last",
    ):
        """Merge a list of sorted DataFrame or Series objects.

        Dataframes/Series in objs list MUST be pre-sorted by columns
        listed in `keys`, or by the index (if `by_index=True`).

        Parameters
        ----------
        objs : list of DataFrame or Series
        keys : list, default None
            List of Column names to sort by. If None, all columns used
            (Ignored if `by_index=True`)
        by_index : bool, default False
            Use index for sorting. `keys` input will be ignored if True
        ignore_index : bool, default False
            Drop and ignore index during merge. Default range index will
            be used in the output dataframe.
        ascending : bool, default True
            Sorting is in ascending order, otherwise it is descending
        na_position : {‘first’, ‘last’}, default ‘last’
            'first' nulls at the beginning, 'last' nulls at the end

        Returns
        -------
        A new, lexicographically sorted, DataFrame/Series.
        """

>       warnings.warn(
            "merge_sorted is deprecated and will be removed in a "
            "future release.",
            FutureWarning,
        )
E       FutureWarning: merge_sorted is deprecated and will be removed in a future release.

../../../miniconda3/envs/gdf/lib/python3.8/site-packages/cudf/core/reshape.py:795: FutureWarning
```

</details>

We intentionally treat warnings as failures in CI to catch such warnings. @madsbk is this something we should fix or should that test be changed or removed?

--- a/dask_cuda/tests/test_proxy.py
+++ b/dask_cuda/tests/test_proxy.py
@@ -576,15 +576,6 @@ def test_einsum_of_proxied_cupy_arrays():
     assert all(res1.flatten() == res2.flatten())
 
 
-def test_merge_sorted_of_proxied_cudf_dataframes():
-    cudf = pytest.importorskip("cudf")
-
-    dfs = [cudf.DataFrame({"a": range(10)}), cudf.DataFrame({"b": range(10)})]
-    got = cudf.merge_sorted(proxify_device_objects(dfs, {}, []))
-    expected = cudf.merge_sorted(dfs)
-    assert_frame_equal(got.to_pandas(), expected.to_pandas())
-
-
 @pytest.mark.parametrize(
     "np_func", [np.less, np.less_equal, np.greater, np.greater_equal, np.equal]
 )

